rtl/eclrtl/rtlfield.cpp

```diff
@@ -2732,7 +2732,7 @@ extern bool ECLRTL_API hasTrailingFileposition(const RtlFieldInfo * const * fiel
         return false;
     while (*fields)
         fields++;
-    return (fields[-1]->type->getType() == type_filepos);
+    return (fields[-1]->type->getType() == type_filepos);  // MORE - is this right? Not sure that this is created any more?
```

Comment:
Created in hqlsource.cpp(527)

Reply:
It creates a RtlSwapIntTypeInfo, but field type is type_filepos

Reply:
Ok I will update the comment to that effect.